mantle/cmd/ore/gcloud/upload.go

```diff
@@ -40,21 +38,19 @@ var (
 
 	uploadBucket    string
 	uploadImageName string
-	uploadBoard     string
 	uploadFile      string
 	uploadFedora    bool
 	uploadForce     bool
 	uploadWriteUrl  string
 )
 
 func init() {
-	build := sdk.BuildRoot()
-	cmdUpload.Flags().StringVar(&uploadBucket, "bucket", "gs://users.developer.core-os.net", "gs://bucket/prefix/ prefix defaults to $USER")
-	cmdUpload.Flags().StringVar(&uploadImageName, "name", "", "name for uploaded image, defaults to COREOS_VERSION")
-	cmdUpload.Flags().StringVar(&uploadBoard, "board", "amd64-usr", "board used for naming with default prefix only")
-	cmdUpload.Flags().StringVar(&uploadFile, "file",
-		build+"/images/amd64-usr/latest/coreos_production_gce.tar.gz",
-		"path_to_coreos_image (build with: ./image_to_vm.sh --format=gce ...)")
+	cmdUpload.Flags().StringVar(&uploadBucket, "bucket", "", "gs://bucket/prefix/")
+	cmdUpload.MarkFlagRequired("bucket")
+	cmdUpload.Flags().StringVar(&uploadImageName, "name", "", "name for uploaded image")
+	cmdUpload.MarkFlagRequired("name")
+	cmdUpload.Flags().StringVar(&uploadFile, "file", "", "path to image file to upload")
```

Comment:
`path to .tar.gz file to upload` for clarity.

Reply:
updated to `path to image .tar.gz file to upload`